examples/create-react-app/README.md

```diff
@@ -0,0 +1,8 @@
+## Example: create-react-app
+
+This shows the usage of CMS design system components in a [create-react-app](https://create-react-app.dev/docs/getting-started/) project.
+
+## Getting started
+
+1. Install packages: `yarn install`
+1. Run the application: `yarn start`
```

Comment:
We should probably have some more information here. Specifically a step for installing the CMSDS as a dependancy, using it in our CSS, setting up font/image paths, etc